samples/subsys/display/cfb/src/main.c

```diff
@@ -52,25 +56,29 @@ int main(void)
 	}
 
 	printf("x_res %d, y_res %d, ppt %d, rows %d, cols %d\n",
-	       cfb_get_display_parameter(dev, CFB_DISPLAY_WIDTH),
-	       cfb_get_display_parameter(dev, CFB_DISPLAY_HEIGH),
+	       x_res,
+	       y_res,
 	       ppt,
 	       rows,
 	       cfb_get_display_parameter(dev, CFB_DISPLAY_COLS));
 
+	cfb_framebuffer_invert(dev);
+
+	cfb_set_kerning(dev, 3);
+
 	while (1) {
-		for (int i = 0; i < rows; i++) {
+		for (int i = 0; i < MIN(x_res, y_res); i++) {
 			cfb_framebuffer_clear(dev, false);
 			if (cfb_print(dev,
 				      "0123456789mMgj!\"§$%&/()=",
-				      0, i * ppt)) {
+				      i, i)) {
 				printf("Failed to print a string\n");
 				continue;
 			}
 
 			cfb_framebuffer_finalize(dev);
 #if defined(CONFIG_ARCH_POSIX)
-			k_sleep(K_MSEC(100));
+			k_sleep(K_MSEC(20));
```

Comment:
I changed the animation that moved 8 dots per 1-frame=(1-loop) to move 1 dot per frame. 
https://github.com/zephyrproject-rtos/zephyr/pull/58366/files#diff-4004fafa8ea903cfd63558c4a188accf91d4cd9db23ca2d3bca269067c8cee8aR74
(ppt is 8)

As a result, the animation eight times slow, so I adjusted the time.
(When this sample was made, drawing only every 8 dots in the vertical direction was possible. Now, It can render by any position.)